ta/ta.mk

```diff
@@ -41,6 +41,7 @@ base-prefix :=
 incdirs-host := $(filter-out lib/libutils%, $(incdirs$(sm)))
 incfiles-extra-host := lib/libutils/ext/include/compiler.h
 incfiles-extra-host += lib/libutils/ext/include/util.h
+incfiles-extra-host += lib/libutils/ext/include/types_ext.h
```

Comment:
It depends if we include `types_ext.h` or if we change to include `stdint.h` instead in `utee_defines.h` above.